### ISSUE TITLE
Game Exception with All Spaces Filled (unsure)

### ISSUE THREAD

**Issue by PCBoyGames** [N3xtery/MAS-Submod-Durak#1]
I think this one is being caused by all six of the spaces that are used being taken up and trying to add another card into another space. But since it has only happened just this once, I am not sure.

However, thank you, Monika. Not only was I really unlucky right before this exception, but then you proceeded to push the boundaries of what your own code can handle.

I'm sorry, but an uncaught exception occurred.

While running game code:
  File "game/submods/Durak/durak.rpy", line 106, in <module>
    $ store.durak.game.game_loop()
  File "game/submods/Durak/durak.rpy", line 432, in game_loop
    self.monika_attack()
  File "game/submods/Durak/durak.rpy", line 1160, in monika_attack
    self.say_quip(self.QUIPS_PLAYER_DRAWS_CARDS[int(round(self.discardpiles_size/2.0-0.6))])
IndexError: tuple index out of range

-- Full Traceback ------------------------------------------------------------

Full traceback:
  File "C:/Users/patri/Downloads/submods/MonikAI_v20/MonikA.I/script-ch30.rpyc", line 2141, in script call
  File "C:/Users/patri/Downloads/submods/MonikAI_v20/MonikA.I/event-handler.rpyc", line 3119, in script call
  File "C:/Users/patri/Downloads/submods/MonikAI_v20/MonikA.I/submods/Durak/durak.rpyc", line 91, in script call
  File "C:/Users/patri/Downloads/submods/MonikAI_v20/MonikA.I/submods/Durak/durak.rpyc", line 106, in script
  File "C:\Games\Doki Doki Literature Club\renpy\ast.py", line 814, in execute
    renpy.python.py_exec_bytecode(self.code.bytecode, self.hide, store=self.store)
  File "C:\Games\Doki Doki Literature Club\renpy\python.py", line 1719, in py_exec_bytecode
    exec bytecode in globals, locals
  File "game/submods/Durak/durak.rpy", line 106, in <module>
    $ store.durak.game.game_loop()
  File "game/submods/Durak/durak.rpy", line 432, in game_loop
    self.monika_attack()
  File "game/submods/Durak/durak.rpy", line 1160, in monika_attack
    self.say_quip(self.QUIPS_PLAYER_DRAWS_CARDS[int(round(self.discardpiles_size/2.0-0.6))])
IndexError: tuple index out of range

Windows-8-6.2.9200
Ren'Py 6.99.12.4.2187
Monika After Story 0.12.15

**Comment by N3xtery** [comment]
Thanks, and you were right about the reason. Will be fixed in v1.0.2
